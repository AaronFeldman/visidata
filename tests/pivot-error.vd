sheet	col	row	keystrokes	input	comment
			o	sample_data/y77d-th95.json.gz	open file
y77d-th95	:@computed_region_nnqa_25f4		!		toggle current column as a key column
y77d-th95	reclat		%		set type of current column to float
y77d-th95	reclat		+	avg	add aggregator to current column
y77d-th95	reclong		+	avg	add aggregator to current column
y77d-th95	reclong		%		set type of current column to float
y77d-th95	fall		W		Pivot the current column into a new sheet
