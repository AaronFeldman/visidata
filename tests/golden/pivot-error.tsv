key1	qty_sum_foo	qty_sum_baz	qty_sum_bar	Total_qty_sum	Total_count
2018-10-20	0	0	0	0	1
2018-07-27	0	256	0	320	3
2017-12-25	0	0	16	16	1
2016-01-01	1	0	0	3	2
date(None)	0	36	0	36	2
Expecting ':' delimiter: line 1 column 34 (char 33)	0	0	0	0	1
